### PR TITLE
Fix french

### PR DESCRIPTION
**Pull request by mariogamer2**

### CODE REVIEW

File: gui/source/languages/french.h [hunk 1]
```diff
@@ -105,7 +105,7 @@ static const char *const lang_FR[STR_MAX] = {
     "Afficher les couleurs de l'arc-en-ciel clignotante dans",                            // "See rainbow colors glowing in"
     "la LED de notification.",                                    // "the Notification LED."
      
-    "Set to TWL to get rid of lags in some games.",             // "Set to TWL to get rid of lags in some games."
+    "Définis à TWL pour se débarrasser de lags de certains jeux.",             // "Set to TWL to get rid of lags in some games."
```
Comment (maorninja) [inline gui/source/languages/french.h]: The last "de" should be replaced with "dans"

**Comment by mariogamer2** [comment]
Wait a little before merging , I think I can add the new settings message.

File: gui/source/languages/french.h (hunk 1 above)
Comment (mariogamer2) [inline gui/source/languages/french.h]: @maorninja Ah thank I was not sure about this :)

Comment (maorninja) [inline gui/source/languages/french.h]: You're welcome! :)

**Comment by mariogamer2** [comment]
Finally I'll just make this. You can merge.

**Pull request merged**